web/src/user_profile.js

```diff
@@ -727,6 +739,31 @@ function get_human_profile_data(fields_user_pills) {
     return new_profile_data;
 }
 
+function get_current_values(user_id) {
+    const person = people.maybe_get_user_by_id(user_id);
+    let current_values = {};
+    if (person.is_bot) {
+        current_values = dialog_widget.get_current_values(
+            $("#bot-edit-form").find("input, select, textarea, button"),
```

Comment:
Can we remove this `if-else` condition by just passing the form element in this function instead of `user_id`?